tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHttpManagerExtensions.cs

```diff
@@ -171,6 +171,21 @@ public static void AddMockHandlerContentNotFound(this MockHttpManager httpManage
             return handler;
         }
 
+        public static MockHttpMessageHandler CreateSuccessfulClientCredentialTokenResponseWithExpiry(
```

Comment:
Code duplication - this is alraedy present in `CreateSuccessfulClientCredentialTokenResponseMessage`. Why not just enhance the existing one?

Reply:
I was thinking this was too specific to add to a general used by many tests but on second thought it is the better option to enhance the existing one